curtsies/curtsieskeys.py

```diff
@@ -112,4 +112,26 @@
   (b"\x1b[[D", u'<F4>'),
   (b"\x1b[[E", u'<F5>'),
   # cool-RR says the rest were good, see issue #99
+
+  #reported by alethiophile see issue #119
+  (b"\x1b[1;3C", u'<Alt-RIGHT>'),      #alt-right
+  (b"\x1b[1;3B", u'<Alt-DOWN>'),       #alt-down
+  (b"\x1b[1;3D", u'<Alt-LEFT>'),       #alt-left
+  (b"\x1b[1;3A", u'<Alt-UP>'),         #alt-up
+  (b"\x1b[5;3~", u'<Alt-PAGEUP>'),     #alt-pageup
+  (b"\x1b[6;3~", u'<Alt-PAGEDOWN>'),   #alt-pagedown
+  (b"\x1b[1;3H", u'<Alt-HOME>'),       #alt-home
+  (b"\x1b[1;3F", u'<Alt-END>'),        #alt-end
+
+  (b"\x1b[1;2C", u'<Shift-RIGHT>'),    #shift-right
```

Comment:
I'd say these are so obvious that we dont' need comments for them, for most of the comments in this file there's some extra information to convey.